Source/WebCore/platform/mediastream/libwebrtc/LibWebRTCProviderCocoa.cpp

```diff
@@ -102,11 +102,7 @@ void LibWebRTCProviderCocoa::setActive(bool value)
 
 bool WebRTCProvider::webRTCAvailable()
 {
-#if PLATFORM(IOS) || PLATFORM(VISION)
```

Comment:
I suggest you actually just check for simulator, rather than changing the behavior on iOS/VisionOS:

#if (PLATFORM(IOS) || PLATFORM(VISION)) && !PLATFORM(IOS_FAMILY_SIMULATOR)